xml_inscriptions/transcribed/MD.Balt.JHU.L.BS.136.xml

```diff
@@ -184,16 +199,28 @@
                   </name>
                   <lb n="2"/>
                   <name type="cognomen" key="Lucifer">Luciferi</name>
-               </persName>, <expan><abbr>Ca<choice><corr>u</corr><sic>e</sic></choice>pion</abbr><ex>iana</ex></expan>.
+               </persName>, <expan><abbr>Ca<choice><corr>e</corr><sic>e</sic></choice>pion</abbr><ex>iana</ex></expan>.
```

Comment:
Something has gone wrong with this change; you're now correcting the "e" to an "e", which seems redundant. Just eliminate all of the choice markup, no?